examples/account_management/get_account_hierarchy.py

```diff
@@ -88,6 +88,11 @@ def main(client, login_customer_id=None):
         unprocessed_customer_ids = [seed_customer_id]
         customer_ids_to_child_accounts = dict()
         root_customer_client = None
+        
+        # login_customer_id is required to query child accounts that are not
+        # directly accessible to the OAuth Credentials
+        client.login_customer_id = str(seed_customer_id)
+        googleads_service = client.get_service("GoogleAdsService")
```

Comment:
Seems like the first `googleads_service` initialization on line 43 can be removed, then line 79 can be updated to use customer service: 

```py

customer_id = customer_service.parse_customer_path(
    customer_resource_name
)["customer_id"]
```